engine/src/main/java/io/zeebe/engine/nwe/behavior/BpmnEventSubscriptionBehavior.java

```diff
@@ -32,7 +34,12 @@
 
 public final class BpmnEventSubscriptionBehavior {
 
+  private static final String NO_WORKFLOW_FOUND_MESSAGE =
+      "Expected to create an instance of workflow with key '%d', but no such workflow was found";
+  private static final String NO_TRIGGERED_EVENT_MESSAGE = "No triggered event for workflow '%d'";
```

Comment:
We should align the error messages with the existing ones (e.g. `Expected ... but ... [context: ...]`.